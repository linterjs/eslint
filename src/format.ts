import {
  LinterAdapterFormatInput,
  LinterAdapterFormatOutput
} from "@linter/core";

export function format({
  filePath,
  text
}: LinterAdapterFormatInput): LinterAdapterFormatOutput {
  // TODO: Format all the things...
  return {};
}
