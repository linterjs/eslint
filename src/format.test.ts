import { format } from "./format";

describe("Format", () => {
  test("Format", () => {
    expect(format({ text: 'var foo = "bar"' })).toMatchSnapshot();
  });
});
