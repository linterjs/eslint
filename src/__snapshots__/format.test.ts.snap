// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format Format 1`] = `"var foo = \\"bar\\""`;
